clientconn.go

```diff
@@ -1204,8 +1204,11 @@ func (ac *addrConn) createTransport(backoffNum int, addr resolver.Address, copts
 	// 2. the internal.HealthCheckFunc is set by importing the grpc/healthcheck package,
 	// 3. a service config with non-empty healthCheckConfig field is provided,
 	// 4. the current load balancer allows it.
+	if !ac.cc.dopts.disableHealthCheck && healthCheckConfig != nil && ac.scopts.HealthCheckEnabled && internal.HealthCheckFunc == nil {
+		grpclog.Errorf("grpc/health package has not been imported to enable health checking.")
```

Comment:
Use `Error` instead of `Errorf` unless you're using args (you can run into problems if you have a literal "%" in your output).